maplibre_gl_web/pubspec.yaml

```diff
@@ -14,15 +14,15 @@ flutter:
       web:
         pluginClass: MapLibreMapPlugin
         fileName: maplibre_gl_web.dart
-
 dependencies:
   flutter:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
   image: ^4.0.17
   js: ">=0.6.7 <0.8.0"
-  maplibre_gl_platform_interface: ^0.20.0
+  maplibre_gl_platform_interface:
+    path: ../maplibre_gl_platform_interface
```

Comment:
Changing this file doesn't seem in scope for the PR.

Reply:
The project uses melos to orchestrate its packages.
```console
dart pub global activate melos
melos bootstrap
```
This will resolve the dependencies so that no changes to pubspec.yaml are necessary.